DOCSP-33302 Update Example Projects Page 9/27/23 (#3030)
mongodb/docs-realm: source/example-projects.txt

      - `.NET <https://github.com/mongodb/template-app-maui-todo>`__
      - `React Native <https://github.com/mongodb/template-app-react-native-todo>`__
      - `Swift <https://github.com/mongodb/template-app-swiftui-todo>`__

  * - Split Data into Multiple Realms  

>     - Build a Netflix-like app for browsing movies from the Mflix sample dataset. 

Reviewer: ```suggestion
    - Build a Netflix-like app for browsing movies from MongoDB's Mflix sample dataset. 
```
same below

>       Use multiple Realms to only allow registered users to sync and manage movies in “My List”.

Reviewer: ```suggestion
      Use multiple realms to only allow registered users to sync and manage movies in “My List”.
```
since it's referring to the file/concept, not the product

same below

>       Use multiple Realms to only allow registered users to sync and manage movies in “My List”.

Reviewer: ```suggestion
      Use multiple Realms to only allow registered users to sync and manage movies in their own private lists.
```
don't love it, but something to communicate that it isn't a shared list / the app is enforcing user-based permissions ("A user will be able to read and write to their own document (i.e. when PrivateContent.userId === <App User ID>), but not anyone else's.")

same below

Author: Makes sense!
